### ISSUE TITLE
The notifications section is not openable

### ISSUE THREAD

**Issue by an2riy** [Blarc/gitlab-template-lint-plugin#40]
Hello.
When the plugin is enabled this section is not loaded: **File | Settings | Appearance & Behavior | Notifications**
![image](https://user-images.githubusercontent.com/8819844/201035897-4ee109d0-e0cb-4106-88e7-32c742643d29.png)
Could be related to:
```cannot create configurable component

java.lang.NullPointerException: Cannot invoke "String.compareTo(String)" because the return value of "Object.toString()" is null
	at com.intellij.notification.impl.ui.NotificationsConfigurablePanel$NotificationsTreeTableModel.lambda$new$0(NotificationsConfigurablePanel.java:330)
	at java.base/java.util.TimSort.binarySort(TimSort.java:296)
	at java.base/java.util.TimSort.sort(TimSort.java:239)
	at java.base/java.util.Arrays.sort(Arrays.java:1307)
	at java.base/java.util.ArrayList.sort(ArrayList.java:1721)
	at com.intellij.notification.impl.ui.NotificationsConfigurablePanel$NotificationsTreeTableModel.<init>(NotificationsConfigurablePanel.java:325)
	at com.intellij.notification.impl.ui.NotificationsConfigurableUi.createNotificationsList(NotificationsConfigurableUi.kt:78)
	at com.intellij.notification.impl.ui.NotificationsConfigurableUi.<init>(NotificationsConfigurableUi.kt:26)
	at com.intellij.notification.impl.NotificationsConfigurable.createUi(NotificationsConfigurable.java:49)
	at com.intellij.notification.impl.NotificationsConfigurable.createUi(NotificationsConfigurable.java:29)
	at com.intellij.openapi.options.ConfigurableBase.createComponent(ConfigurableBase.java:57)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.createComponent(ConfigurableWrapper.java:169)
	at com.intellij.openapi.options.ex.ConfigurableCardPanel.lambda$createConfigurableComponent$4(ConfigurableCardPanel.java:116)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:941)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:68)
	at com.intellij.openapi.options.ex.ConfigurableCardPanel.createConfigurableComponent(ConfigurableCardPanel.java:112)
	at com.intellij.openapi.options.ex.ConfigurableCardPanel.create(ConfigurableCardPanel.java:60)
	at com.intellij.openapi.options.newEditor.ConfigurableEditor$1.create(ConfigurableEditor.java:57)
	at com.intellij.openapi.options.newEditor.ConfigurableEditor$1.create(ConfigurableEditor.java:54)
	at com.intellij.ui.CardLayoutPanel.createValue(CardLayoutPanel.java:75)
	at com.intellij.ui.CardLayoutPanel.select(CardLayoutPanel.java:103)
	at com.intellij.ui.CardLayoutPanel.lambda$selectLater$0(CardLayoutPanel.java:135)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:21)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.openapi.application.impl.ApplicationImpl$3.run(ApplicationImpl.java:513)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:75)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:118)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:42)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:779)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:749)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:918)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:450)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:449)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:624)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:447)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:121)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1080)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:712)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:453)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1672)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1630)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.java:141)
	at com.github.blarc.gitlab.template.lint.plugin.GitlabLintBundle.openPluginSettings(GitlabLintBundle.kt:30)
	at com.github.blarc.gitlab.template.lint.plugin.notifications.NotificationAction$Companion$settings$1.invoke(Notification.kt:85)
	at com.github.blarc.gitlab.template.lint.plugin.notifications.NotificationAction$Companion$settings$1.invoke(Notification.kt:83)
	at com.github.blarc.gitlab.template.lint.plugin.notifications.NotifierKt.sendNotification$lambda-1$lambda-0(Notifier.kt:32)
	at com.intellij.openapi.project.DumbAwareAction$SimpleDumbAwareAction.actionPerformed(DumbAwareAction.java:82)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:315)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:294)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:337)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:294)
	at com.intellij.openapi.fileEditor.impl.IdeUiServiceImpl.performActionDumbAwareWithCallbacks(IdeUiServiceImpl.java:109)
	at com.intellij.notification.Notification.fire(Notification.java:288)
	at com.intellij.notification.impl.NotificationsManagerImpl.lambda$createAction$11(NotificationsManagerImpl.java:841)
	at com.intellij.ui.components.labels.LinkLabel.doClick(LinkLabel.java:174)
	at com.intellij.ui.components.labels.LinkLabel.doClick(LinkLabel.java:388)
	at com.intellij.ui.components.labels.LinkLabel$MyMouseHandler.mouseReleased(LinkLabel.java:361)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6648)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3392)
	at java.desktop/java.awt.Component.processEvent(Component.java:6413)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2802)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:754)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:752)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:751)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:918)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:840)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:763)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:450)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:449)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:624)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:447)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:881)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:493)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

The system info:
```
PhpStorm 2022.2.3
Build #PS-222.4345.15, built on October 5, 2022
Subscription is active until June 8, 2023.
Runtime version: 17.0.4.1+7-b469.62 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.15.0-52-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 5120M
Cores: 12
Registry:
    editor.focus.mode.color.light=5b595c
    ide.editor.tab.selection.animation=true
    debugger.watches.in.variables=false
    debugger.new.tool.window.layout=true
    editor.caret.width=1
    run.processes.with.pty=true
    vcs.new.widget=true
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=5b595c

Non-Bundled Plugins:
    ski.chrzanow.foldableprojectview (1.1.3)
    pronskiy.elephpant (0.1.1)
    org.elasticsearch4idea (2022.3.0-222)
    name.kropp.intellij.makefile (222.4345.14)
    intellij-awk (0.3.5)
    indent-rainbow.indent-rainbow (2.0.2)
    de.femtopedia.diffplugin (2.1.8)
    com.ultrahob.zerolength.plugin (0.95)
    com.intellij.properties (222.4345.14)
    com.hjf.redis.manager.redis-manager (1.4.1)
    com.alayouni.ansiHighlight (22.2.3)
    String Manipulation (9.6.1)
    GrepConsole (12.15.211.6693.1)
    zielu.gittoolbox (212.9.9)
    uk.co.ben-gibson.remote.repository.mapper (4.2.4)
    mobi.hsz.idea.gitignore (4.4.2)
    com.mallowigi (73.0.0)
    de.espend.idea.php.phpunit (6.2)
    de.espend.idea.php.annotation (8.2.3)
    com.kalessil.phpStorm.phpInspectionsEA (4.0.7.1)
    com.magento.idea.magento2plugin (4.4.0)
    lv.midiana.misc.phpstorm-plugins.deep-keys (2022.08.03.001)
    izhangzhihao.rainbow.brackets (2022.3.3.1-ij)
    com.chrisrm.idea.MaterialThemeUI (7.14.2)
    ir.msdehghan.plugins.ansible (0.92)
    fr.adrienbrault.idea.symfony2plugin (2022.1.235)
    com.github.blarc.gitlab-template-lint-plugin (1.1.1)
    org.jetbrains.plugins.node-remote-interpreter (222.4345.14)
    ru.adelf.idea.dotenv (2022.2)
    pro.bashsupport (3.0.1.222)

Current Desktop: ubuntu:GNOME
```

Please take a look.

**Comment by Blarc** [comment]
Thanks for the report, I will look into it.

**Comment by Blarc** [comment]
I have released a new version 1.2.2. Can you please check, if the bug was resolved? @an2riy

**Comment by an2riy** [comment]
> I have released a new version 1.2.2. Can you please check, if the bug was resolved? @an2riy

Confirmed. Thanks
